sagemaker-fundamentals/create-training-job/container_kms/train.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+import boto3
+    
+import base64
+from cryptography.fernet import Fernet
+import os
+import json
+import pickle
+
+import pprint
+
+# init a boto3 session
+sess = boto3.Session(
+    aws_access_key_id=os.environ['ACCESS_KEY'],
+    aws_secret_access_key=os.environ['SECRET_KEY'],
+    region_name=os.environ['REGION_NAME']
```

Comment:
Does SageMaker not expose the meta data endpoints?

Reply:
It is not straightforward to do so inside the container, you can use [ContainerProvider](https://github.com/boto/botocore/blob/develop/botocore/credentials.py#L1819). But I think a standalone notebook should be used to explain ContainerProvider because it can be used in much wider context